api/services/ChatService.js

```diff
@@ -13,11 +13,7 @@ function getChats() {
 }
 
 function addUserMessage(chat) {
-  if (!chat.username) {
-    chat.username = 'Anonymous';
-  }
-  chat.type = 'user';
-  chat.time = new Date(chat.time).toISOString()
+  chat.time = new Date().toISOString();
```

Comment:
With actions like `skip` the real source of truth is when the server decides to skip the song. When it comes to chat, chat occurs at the moment the user hits enter